src/watchmaker/exceptions/__init__.py

```diff
@@ -1,18 +1,127 @@
+import datetime
 import logging
+import os
 import sys
 
 
-def SystemFatal(msg):
-    """
-    This is a temporary exception handler for the project.
+class LogHandler:
```

Comment:
My thinking on why it is odd to me is that with this in the exceptions module, I would expect to use `raise LogHandler()`, but that doesn't seem like what we want to do.